Qubit Server/dev/src/main/java/org/labrad/qubits/QubitContext.java

```diff
@@ -562,22 +562,34 @@ public void sram_dual_block_delay(@Accepts("v[ns]") double delay) {
           + "should be deconvolved (default: true)."
           + "If deconvolve=false, the data can be specified as DAC-ready"
           + "I and Q integers.")
-  public void sram_iq_data(@Accepts({"s", "ss"}) Data id,
-      @Accepts("*c") Data vals) {
+  public void sram_iq_data(
+      @Accepts({"s", "ss"}) Data id,
+      @Accepts("*c") Data vals
+  ) {
     sram_iq_data(id, vals, true);
-    sramDirty = true;
```

Comment:
it gets set by the method that we called on the previous line. Incidently, all this boilerplate goes in scala because it has optiona parameters with default values.